airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/utils.py

```diff
@@ -323,6 +329,9 @@ def transform_data(records: List) -> Iterable[Mapping]:
         if "variables" in record:
             record = transform_variables(record)
 
+        if "pivotValues" in record:
+            record = transform_pivot_values(record)
```

Comment:
This 'if' implies that one of the primary keys for the stream can be null.  Is it the case that at least the other PK, end_date, will always be non-null?